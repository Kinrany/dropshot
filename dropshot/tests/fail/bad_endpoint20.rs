// Copyright 2023 Oxide Computer Company

#![allow(unused_imports)]

use dropshot::endpoint;
use dropshot::HttpError;
use dropshot::HttpResponseOk;
use dropshot::UntypedBody;

#[endpoint {
    method = GET,
    path = "/test",
    request_body_max_bytes = true,
}]
async fn bad_request_body_max_bytes(
    _rqctx: RequestContext<()>,
    param: UntypedBody,
) -> Result<HttpResponseOk<()>, HttpError> {
    Ok(HttpResponseOk(()))
}

fn main() {}
