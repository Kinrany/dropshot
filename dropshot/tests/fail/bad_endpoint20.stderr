error: expected u64, but found `true`
  --> tests/fail/bad_endpoint20.rs:13:30
   |
13 |     request_body_max_bytes = true,
   |                              ^^^^
